### PR TITLE
Add paywall script

### PR DESCRIPTION
**Pull request by davidyuk**
depends on #25 https://github.com/aeternity/superhero-wallet/pull/490 https://github.com/aeternity/superhero-wallet/pull/485

works better with https://github.com/aeternity/superhero-wallet/pull/487

https://forum.aeternity.com/t/feature-paywall-script-for-content-owners/7045/6
https://p.aeternity.org/projects/superhero-launch/work_packages/1427/activity

Screenshot 2020-10-05 at 16 52 20

### CODE REVIEW

**Comment by davidyuk** [comment]
As I see, the only thing that blocks merging of this is https://github.com/aeternity/superhero-wallet/pull/485

**Comment by mradkov** [comment]
> As I see, the only thing that blocks merging of this is [aeternity/superhero-wallet#485](https://github.com/aeternity/superhero-wallet/pull/485)

It's merged already.

**Review by mradkov — APPROVED** [review]

**Comment by mradkov** [comment]
Can you also update the documentation with an example of how this can be used? Thanks

File: README.md [hunk 1]
```diff
@@ -1,117 +1,94 @@
...
 ```html
-<div class="small">Donate</div>
-<script src="https://unpkg.com/@aeternity/superhero-button/dist/superhero-button.styles.js"></script>
+<div class="my-button">Donate</div>
 <script type="text/javascript">
-  superheroButton('.small', { size: 'small', account: 'ak_... or .chain name' });
+  superheroButton('.my-button', {
+    size: 'large',
+    account: 'example.chain',
+    url: 'http://example.com',
```
Comment (mradkov) [inline README.md]:
```suggestion
    url: 'https://example.com',
```

I suggest we have the documentation with `https` examples only.

File: README.md [hunk 2]
```diff
@@ -1,117 +1,94 @@
...
 
-<img width="140" alt="Screenshot 2020-05-13 at 9 55 54" src="https://user-images.githubusercontent.com/13139371/81780943-0387ee00-9500-11ea-8108-2e5939821a7b.png">
+#### Example
 
 ```html
-<div class="large">Donate</div>
-<script src="https://unpkg.com/@aeternity/superhero-button/dist/superhero-button.styles.js"></script>
 <script type="text/javascript">
-  superheroButton('.large', { size: 'large', account: 'ak_... or .chain name' });
+  superheroButton.ensurePayed({ url: 'http://example.com' });
```
Comment (mradkov) [inline README.md]:
```suggestion
  superheroButton.ensurePayed({ url: 'https://example.com' });
```

**Pull request merged**